app/static/tags/securitygroup/securitygroupform.tag

```diff
@@ -0,0 +1,99 @@
+            <div class="help-block" ng-show="securitygroupForm.sgname.$dirty && securitygroupForm.sgname.$invalid">
+                <i class="fa fa-warning"></i> {{msg.template_name_invalid}}
+            </div>
+        </div>
+        <!-- .col-sm-9 -->
+
+    </div>
+
+    <div class="form-group" ng-class="{ 'has-error': securitygroupForm.description.$dirty && securitygroupForm.description.$invalid }">
+        <label class="col-sm-3 control-label" for="description">{{msg.description_label}}</label>
+
+        <div class="col-sm-9">
+            <input type="text" class="form-control" name="description" ng-model="securitygroup.description" ng-maxlength="1000" id="description" placeholder="{{msg.template_form_description_placeholder}}">
+            <div class="help-block" ng-show="securitygroupForm.description.$dirty && securitygroupForm.description.$invalid">
+                <i class="fa fa-warning"></i> {{msg.template_description_invalid}}
+            </div>
+        </div>
+        <!-- .col-sm-9 -->
+    </div>
+    <div class="form-group" ng-class="{ 'has-error': securitygroupForm.securitygroup_publicInAccount.$dirty && securitygroupForm.securitygroup_publicInAccount.$invalid }">
+            <label class="col-sm-3 control-label" for="securitygroup_publicInAccount">{{msg.public_in_account_label}}</label>
+            <div class="col-sm-9">
+                <input type="checkbox" name="securitygroup_publicInAccount" id="securitygroup_publicInAccount" ng-model="securitygroup.public">
+            </div>
+    </div>
+
+    <div class="form-group">
+        <label class="col-sm-3 control-label">Ports</label>
+        <div class="col-sm-8 col-sm-offset-1">
+            <form class="form-horizontal" role="form" name="tmpsecportForm">
+                <div class="form-group" style="padding-top: 10px;" ng-class="{ 'has-error': tmpsecportForm.cidr.$dirty && tmpsecportForm.cidr.$invalid }">
+                    <label class="col-sm-3 control-label">CIDR: </label>
+                    <div class="col-sm-9">
+                        <input type="text" class="form-control" name="cidr" ng-model="tmpsecport.cidr" placeholder = "0.0.0.0/0" id="cidr" ng-pattern="/^(([0-9]|[1-9][0-9]|1[0-9]{2}|2[0-4][0-9]|25[0-5])\.){3}([0-9]|[1-9][0-9]|1[0-9]{2}|2[0-4][0-9]|25[0-5])(\/([0-9]|[1-2][0-9]|3[0-2]))$/">
+                        <div class="help-block" ng-show="tmpsecportForm.seccidr.$dirty && tmpsecportForm.seccidr.$invalid">
+                            <i class="fa fa-warning"></i> szar
+                        </div>
+                    </div>
+                </div>
+                <div class="form-group" ng-class="{ 'has-error': tmpsecportForm.port.$dirty && tmpsecportForm.port.$invalid }">
+                    <label class="col-sm-3 control-label">Port: </label>
+                    <div class="col-sm-9">
+                        <input type="text" class="form-control" name="port" placeholder = "4444,5555,6666" ng-model="tmpsecport.port" id="port">
```

Comment:
This list could contain a list of strings too, but that is not validated on the backend neither.